takwimu/templates/profile/profile_detail.html

```diff
@@ -1,169 +1,267 @@
+          </section>
+
         </div>
+      </article>
+
+      <article
+          class="clearfix">
+        <header class="section-contents">
+          <h1>Census</h1>
+        </header>
+        <div class="section-container">
+          <section class="clearfix stat-row">
+            <h4>Population 2016</h4>
+            {% include 'profile/_blocks/_stat_list.html' with stat=demographics.total_population stat_type='number' %}
+          </section>
+          <section class="clearfix stat-row">
+            <div class="column-full"
+                 id="chart-histogram-demographics-population_dist"
+                 data-stat-type="number"
+                 data-chart-title="Population For the last 5 years"></div>
+          </section>
 
-        <div class="column-third">
-          {% include 'profile/_blocks/_stat_list.html' with stat=landscape.public_health_expenditure_of_gdp stat_type='percentage' %}
         </div>
+      </article>
+
+      <article
+          class="clearfix {% if 'demographics' not in selected_sections %}hide{% endif %}">
+        <header class="section-contents">
+          <h1>Education</h1>
+        </header>
+        <div class="section-container">
+          <section class="clearfix stat-row">
+            <div class="column-full"
+                 id="chart-histogram-literacy-literacy"
+                 data-stat-type="scaled-percentage"
+                 data-chart-title="Literate Percentage of population above 15 years"></div>
+          </section>
 
-        <div class="column-third">
-          {% include 'profile/_blocks/_stat_list.html' with stat=landscape.internet_users__of_population stat_type='percentage' %}
         </div>
-      </section>
-    </div>
-    <div class="section-container">
-      <section class="clearfix stat-row">
+      </article>
+      <article
+          class="clearfix {% if 'demographics' not in selected_sections %}hide{% endif %}">
+        <header class="section-contents">
+          <h1>Agriculture</h1>
+        </header>
+        <div class="section-container">
+          <section class="clearfix stat-row">
+            <div class="column-full"
+                 id="chart-histogram-crops-crop"
+                 data-stat-type="number"
+                 data-chart-title="Crop Production Index For the last 10 years"></div>
+          </section>
 
-        <div class="column-third">
-          {% include 'profile/_blocks/_stat_list.html' with stat=landscape.gender_development_index stat_type='number' %}
         </div>
+      </article>
+    {% endblock %}
+  </div>
+
+  {% include 'takwimu/_includes/sections/support-services/button.html' %}
+
+  {% include 'takwimu/_includes/sections/stories/index.html' %}
+
+  <!-- Country profile country select -->
+  {% include 'takwimu/_includes/sections/explore/_index.html' %}
+
+  {% include 'takwimu/_includes/sections/updates.html' %}
+
+  {# Footer #}
+  {% include 'takwimu/_includes/footer.html' %}
```

Comment:
There's an errant whitespace between the updates and footer. Please fix.

Reply:
Fixed